assets/js/src/settings/store/types.ts

```diff
@@ -1,3 +1,50 @@
+type WebHostName =
+  | 'manual'
+  | '1and1'
+  | 'bluehost'
+  | 'df'
+  | 'dreamhost'
+  | 'free'
+  | 'froghost'
+  | 'godaddy'
+  | 'goneo'
+  | 'googleapps'
+  | 'greengeeks'
+  | 'hawkhost'
+  | 'hivetec'
+  | 'hostgator'
+  | 'hosting2go'
+  | 'hostmonster'
+  | 'infomaniak'
+  | 'justhost'
+  | 'laughingsquid'
+  | 'lunarpages'
+  | 'mediatemple'
+  | 'netfirms'
+  | 'netissime'
+  | 'one'
+  | 'ovh'
+  | 'phpnet'
+  | 'planethoster'
+  | 'rochen'
+  | 'site5'
+  | 'siteground'
+  | 'synthesis'
+  | 'techark'
+  | 'vexxhost'
+  | 'vps'
+  | 'webcity'
+  | 'westhost'
+  | 'wpwebhost'
+
+type AmazonSesRegion =
+  | 'us-east-1'
+  | 'us-west-2'
```

Comment:
These two types seems a bit odd to me. When we add a new region or host we need to add it to more places and most certainly forget. 

Does the frontend care if exactly those values came from the backend? Should we just use `string` value here? The same probably applies to `normalizeSettings` where those values are also duplicated...

Reply:
The frontend doesn't care what value is given by the API. These types will only be useful if we decided to write a value of this type manually in TS code, like giving it as prop to a component or writing it during tests for example. if you think that the probability of that is low then we can use `string` instead. What do you think?

Reply:
Yeah, I think we should just use string here.